### ISSUE TITLE
Is it possible to add in "background" image?

### ISSUE THREAD

**Issue by leeseng0629** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#186]
By 'background' image, I means those images without any of my labelled classes.

I had created a model successfully by following this repos' tutorial, and the performance of the model is quite good, but now I try to reduce the false positive (Mostly occurred on those 'background' images). So, I was wondering how to add in those 'background' images and below is what I found,

When I look into the xml_to_csv.py, it tends to ignore those .xml files that without <object> tag (because of the for loop), so, what I'm thinking is to modify it, so it still write those entries into csv file but leave the class, xmin, ymin, xmax, ymax columns, and then convert to .record files,  and finally use the newly generated .record file to train my model.

May I know any of you try this way before? If this cannot work, anyone have any idea to add in 'background' images?

**Comment by Twikkxx** [comment]
Hello, is there any progress/update on this?

**Comment by YonitZall** [comment]
I'm also trying to do the same thing, but I don't know how to write these background images in the csv.... Would love to get also an answer

**Comment by Adnan-annan** [comment]
Hi, Is there any way to add background class to tensorflow object detection API ? and how to label the background class ?

**Comment by yxyangxu** [comment]
The loss suddenly became huge in some steps when training after I add background images into datasets, and finally the loss doesn't converge
